htdocs/css/jquery-ui-1.10.4.custom.css

```diff
@@ -7,10 +7,10 @@
 /* Layout helpers
 ----------------------------------*/
 .ui-helper-hidden {
-	display: none;
+    display: none;
 }
 .ui-helper-hidden-accessible {
-	border: 0;
+    border: 0;
```

Comment:
indentation needs to be updated.

Reply:
That indentation looks correct to me.

Reply:
@intralizee i think @um4r12 means this line 13 is not indented the same as the lines below it